magento/magento2
# Store stock channel table for msi inventory mapping

### Description
Created additional table to store stock channel items for msi inventory mapping.

### Manual testing scenarios
1. Install InventorySales and InventorySalesApi Module and check if 'inventory_stock_channel' table was created.

### Contribution checklist
